colossalai/auto_parallel/tensor_shard/utils/device/get_alpha_beta.py

```diff
@@ -0,0 +1,13 @@
+from get_one_alpha_beta import execute_cmd
```

Comment:
This should be put under the directory `colossalai.device`.

Reply:
`from get_one_alpha_beta import execute_cmd` should be `from .get_one_alpha_beta import execute_cmd` to avoid possible namespace conflict.

Reply:
I move all the files to the `colossalai/device` folder.

Reply:
It is still in `colossalai/auto_parallel/tensor_shard/utils/device/get_alpha_beta.py` ?